TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -83,6 +83,22 @@ const BenchmarkMetricRegex = {
             },
         }
     },
+    Score: {
+        metrics: {
+            "Score":{
+                //Example: ArrayListSortComparatorBench score: 2757481.000000 (2.757M 1483.0%)
+                regex: /ArrayListSort[\s\S\n]*? score: ?(\d*\.?\d*)/,
+                higherbetter: true,
+                units: "kb",
+            },
+            "Uncertainty %":{
+                //Example: uncertainty:   0.8%
+                regex:/uncertainty:   ?(\d*\.?\d*)/,
```

Comment:
Shouldn't hard-code spaces. Please make to follow the convention mentioned at the top of this file for all regexes. Please double check the one for score metric as well.